lib/utils/gestures.js

```diff
@@ -630,12 +630,12 @@ function _findRecognizerByEvent(evName) {
  * This value is checked on first move, thus it should be called prior to
  * adding event listeners.
  *
- * @param {!Node} node Node to set touch action setting on
+ * @param {!EventTarget} node Node to set touch action setting on
  * @param {string} value Touch action value
  * @return {void}
  */
 export function setTouchAction(node, value) {
-  if (HAS_NATIVE_TA) {
+  if (HAS_NATIVE_TA && node instanceof Node) {
```

Comment:
This should be a check for `Element` to ensure the style property.

Reply:
Done (HTMLElement actually)